content/posts/2023/reana-0.9.0.md

```diff
@@ -0,0 +1,161 @@
+to learn more about this feature.
+
+### Launching workflows from external sources using web interface
+
+You can now easily run your analysis workflows from publicly accessible external
+sources, like GitHub, GitLab, digital repositories like Zenodo or any website on
+the web using the new REANA launcher page. You can host your workflows on a
+public location and be able to access and run them on REANA without having to do
+any manual steps.
+
+[![Launch home](/images/reana-0.9.0-launcher.png)](https://reana.cern.ch/launch)
+
+For more information about this new feature please check the dedicated blog post
+about [launching workflows from external sources using web interface](/posts/2022/launching-workflows/).
+
+### Rucio authentication for workflow jobs
+
+[Rucio](https://rucio.cern.ch/) is a scientific data management system used in
+LHC particle physics and related scientific domains. It provides access for
+large volumes of data spread across facilities at multiple institutions.
+
+If your workflow needs to use data that is managed by a Rucio system, you can
+follow the documentation that describes the [Rucio authentication method](https://docs.reana.io/advanced-usage/access-control/rucio/)
+to learn more about it.
+
+### Kerberos authentication for workflow orchestration
+
+Previously, you had to set up Kerberos authentication for each step of your
+workflow, one at a time. You can now easily enable Kerberos authentication
+for your entire workflow by adding just one line in the specification file. This
+makes it much more convenient and efficient. This technique is especially useful
+for workflows that need to access files in restricted storage in multiple steps.
+
+Please check our documentation about [enabling Kerberos support for an entire workflow](https://docs.reana.io/advanced-usage/access-control/kerberos/#setting-kerberos-requirement-for-whole-workflow)
+to learn more about this feature.
+
+### Notifications to inform when critical levels of quota usage is reached
+
+A new notifications feature has been added to REANA web interface to alert users
+when their [quota](https://docs.reana.io/advanced-usage/user-quotas) is
+approaching or has reached its limit. This feature is intended to help users
+stay within their allocated Disk space and CPU consumption limits and inform
+them when deletion of unnecessary workflow runs should be performed in order to
+liberate some resources.
+
+![Quota notifications](/images/reana-0.9.0-notifications.png)
+
+## What's new for the administrators?
+
+### Configure global workspace retention rules
+
+Workspace retention rules are giving users the possibility to remove unwanted files from their workflows' workspaces.
+However, administrators can also define a global workspace file retention period, after which all the files that are not listed as inputs or outputs of a workflow will be deleted, regardless of user-defined workspace retention rules.
```

Comment:
"regardless..." evoked a bit negative feelings. What about simply ending the phrase e.g. "will be deleted".  Or "will be deleted at the latest" or "... by default".